### DISCUSSION TITLE
Not working on older versions

### DISCUSSION THREAD

**Discussion by gailium119** [valinet/ExplorerPatcher#812] [General]
When installing the latest version on 22000.1, explorer just basically remain the same, and the start menu couldn't open.The system is original from uupdump and no system files are changed.

**Comment by valinet** [comment]
Yeah, possible, I haven’t tested against that in a very long time (i.e. more precisely, since early last autumn). Maintaining support for all previous builds is kind of difficult.

**Reply by gailium119** [reply]
So how could i get this working?

**Comment by gailium119** [comment]
Just found out the newest EP version for 22000.1 is 22449.1000.0.18.What did you change in 22000.194.0.20?

**Reply by valinet** [reply]
I don't remember. Quite a few things since October 2021. Checkout some commit around that date, that's why the source code is available. Honestly, quite a lot. And not only changed, also implemented tons of things from scratch. Upgrade to the latest build, 22000.1 was never really supposed to be a shipping build anyway.

**Reply by gailium119** [reply]
Not exactly.I've figured out the files related to explorer.exe on 22000.1 and replaced the ones in 22000.469, restoring a 22000.1 UI on 22000.469
